core/sdk/src/service/service_client.rs

```diff
@@ -170,11 +222,45 @@ impl ServiceClient {
                 return;
             }
         };
+
+        let mut retry_count = 0u32;
         loop {
             debug!("Connecting to WebSocket at {}", path);
             Self::connect_websocket(self, &path, &mut outbound_service_rx, &mut outbound_extn_rx)
                 .await;
+
             debug!("Initialize Ended Abruptly");
+
+            // Exponential backoff with jitter to avoid DOSing the service
+            // Base delay starts at 100ms and caps at 10 seconds
```

Comment:
Reduce the base delay to a lower value for service connection (5m). Goal is to connect to RippleMain ASAP on boot-up use case. Exponential backoff is really useful for re-connect use case.